pay-api/src/pay_api/services/eft_service.py

```diff
@@ -35,9 +36,13 @@ class EftService(DepositService):
     """Service to manage electronic fund transfers."""
 
     def get_payment_method_code(self):
-        """Return EFT as the system code."""
+        """Return EFT as the payment method code."""
         return PaymentMethod.EFT.value
 
+    def get_payment_system_code(self):
+        """Return PAYBC as the system code."""
```

Comment:
Is the AR module considered PAYBC? I thought PAYBC was just credit cards, I could be wrong

Reply:
I might wrong, I looked at pay_api code, the direct_pay, online banking, pad return the paybc system code, so I thought it same for eft.

Reply:
no worries, sounds good